### PR TITLE
update VSS version

### PR DESCRIPTION
**Pull request by reznikmm**

### CODE REVIEW

File: index/vs/vss/vss-22.0.0-20210224.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name = "vss"
+description = "Advanced string and text manipulation with Unicode support"
+version = "22.0.0-20210224"
```
Comment (mosteo) [inline index/vs/vss/vss-22.0.0-20210224.toml:3]: Hi Maxim. If you want to include the date just for informative purposes, the separator according to [semver](https://semver.org/#spec-item-11) should be `+`, as `-` means a pre-22 alpha release.

**Review by mosteo — CHANGES_REQUESTED** [review]

File: index/vs/vss/vss-22.0.0-20210224.toml (hunk 1 above)
Comment (Fabien-Chouteau) [reply index/vs/vss/vss-22.0.0-20210224.toml:3]: We want pre-22 alpha here.

Comment (mosteo) [reply index/vs/vss/vss-22.0.0-20210224.toml:3]: Sometimes I worry too much. Alright then!

**Review by mosteo — APPROVED** [review]

File: index/vs/vss/vss-22.0.0-20210224.toml (hunk 1 above)
Comment (Fabien-Chouteau) [reply index/vs/vss/vss-22.0.0-20210224.toml:3]: It was a good question :)

**Pull request merged**